x/auth/vesting/exported/exported.go

```diff
@@ -11,6 +11,8 @@ import (
 type VestingAccount interface {
 	authexported.Account
 
+	UnspendableCoins(currVestDenomsBalance sdk.Coins, blockTime time.Time) sdk.Coins
```

Comment:
What do we need this function for?

Reply:
This API doesn't make any sense.

Reply:
But that would require passing all coins to the accounts, meaning you'd have to get all coins from the bank store.

Reply:
Yes, upon further review this makes sense. However, I still don't like the API's nomenclature. I'll change it to `LockedCoins(sdk.Coins, time.Time) sdk.Coins`